### PR TITLE
repository

### PR DESCRIPTION
**Pull request by quannb99**
## Related Tickets:
- [#31913](https://edu-redmine.sun-asterisk.vn/issues/31913)

## WHAT this PR does?
- Repository part 1

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by quannb99** [comment]
ready

File: app/Http/Controllers/MarkingController.php [hunk 1]
```diff
@@ -2,55 +2,70 @@
...
+    protected $commentRepo;
+    protected $likeRepo;
+
+    public function __construct(
+        UserRepositoryInterface $userRepositoryInterface,
+        ReviewRepositoryInterface $reviewRepositoryInterface,
+        CommentRepositoryInterface $commentRepositoryInterface,
+        LikeRepositoryInterface $likeRepositoryInterface
+    )
+    {
```
Comment (huyenttm-0901) [inline app/Http/Controllers/MarkingController.php]:
```suggestion
    ) {
```

File: app/Http/Controllers/MarkingController.php [hunk 2]
```diff
@@ -2,55 +2,70 @@
...
-            $review->update([
-                'like_num' => $review->likes_count,
-            ]);
+            $likeNum = $this->reviewRepo->updateLikeNum($id);
         } catch (ModelNotFoundException $e) {
             return redirect()->back()->with('fail_status', trans('msg.find_fail'));
         }
 
-        return $review->likes_count;
+        return $likeNum;
```
Comment (huyenttm-0901) [inline app/Http/Controllers/MarkingController.php]: đặt tên biến có ý nghĩa vào em

File: app/Repositories/Book/BookRepositoryInterface.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Repositories\Book;
+
+interface BookRepositoryInterface
```
Comment (huyenttm-0901) [inline app/Repositories/Book/BookRepositoryInterface.php]: extends baseInterface nhé

**Comment by quannb99** [comment]
ready

**Pull request merged**